test/semantics/select-case.f90

```diff
@@ -0,0 +1,167 @@
+!===--- select-case.f90 - Test select case constraints -------------------===
+!
+! Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+! See https://llvm.org/LICENSE.txt for license information.
+! SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+!
+!===------------------------------------------------------------------------===
```

Comment:
Apparently we don't need this header on test files.

Reply:
I will remove the header.